javascript-modal/main.js

```diff
@@ -0,0 +1,13 @@
+const $modal = document.querySelector('#b1');
+const $survey = document.querySelector('#b2');
+
+const $container2 = document.querySelector('.container2.hide');
+console.log($container2);
```

Comment:
Remove all `console.log`s before submitting your code! They are only for testing / debugging purposes